showcase/src/app/demo/collapsible/collapsible.component.html

```diff
@@ -0,0 +1,91 @@
+	<stark-collapsible 
+			htmlId="id0" 
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_0" 
+			[(isExpanded)]="collapsed[0]"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(0)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'TS']" filesPath="collapsible/collapsible-custom1" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.CUSTOM_ICON">
+	<stark-collapsible
+			htmlId="id1"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[1]"
+			icon="chevron-down"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(1)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<br/>
+	<br/>
+	<stark-collapsible
+			htmlId="id2"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[2]"
+			icon="information"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(2)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<br/>
+	<br/>
+	<stark-collapsible
+			htmlId="id3"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[3]"
+			icon="account-circle"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(3)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'TS']" filesPath="collapsible/collapsible-custom2" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.CUSTOM_OPTIONS">
+	<stark-collapsible
+			htmlId="id4"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_2"
+			subTitleLabel="SHOWCASE.DEMO.COLLAPSIBLE.SUBTITLE"
+			[(isExpanded)]="collapsed[4]"
+			[options]="options1"
+			icon="chevron-down"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(4)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<br/>
+	<br/>
+	<stark-collapsible
+			htmlId="id5"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_2"
+			subTitleLabel="SHOWCASE.DEMO.COLLAPSIBLE.SUBTITLE"
+			[(isExpanded)]="collapsed[5]"
+			[options]="options2"
+			icon="loop"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<br/>
+	<button color="primary" (click)="toggleCollapsible(5)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<br/>
+	<br/>
```

Comment:
Can you add maybe a margin in a SCSS to add more space so it will be more obvious on which collapsible the button will interact with?